.github/workflows/build-container.yml

```diff
@@ -0,0 +1,50 @@
+name: dispatched build-docker-image
+
+on:
+  push:
+    branches:
+      - master
+  repository_dispatch:
+    types:
+      - dispatch-build
+
+jobs:
+  make-date-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      dtag: ${{ steps.mkdatetag.outputs.dtag }}
+    steps:
+    - name: make date tag
+      id: mkdatetag
+      run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
+
+  build:
+    runs-on: ubuntu-latest
+    if: github.repository == 'opensciencegrid/osgvo-docker-pilot'
+    strategy:
+      matrix:
+        repo: ['development', 'testing', 'release']
+
+    steps:
+    - name: checkout osgvo-docker-pilot
+      uses: actions/checkout@v2
+
+    - id: generate-tag-list
+      env:
+        REPO: ${{ matrix.repo }}
+        TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
+      run: |
+        docker_repo=opensciencegrid/osgvo-docker-pilot
+        tag_list=$docker_repo:$REPO,$docker_repo:$REPO-$TIMESTAMP
```

Comment:
do we really need the docker repo name in the tag?  Previously the build-and-push-docker-images step worked specifying `repository` and the unqualified tag names